stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -163,6 +168,11 @@ spec:
         secret:
           secretName: {{ .Values.backend.es.tls_secret | default ( print ( include "fluent-bit.fullname" . ) "-es-tls-secret" ) | quote }}
 {{- end }}
+{{- if and .Values.backend.type "stackdriver" .Values.backend.stackdriver.google_service_credentials }}
+      - name: gcp-secret
```

Comment:
Can we name this secret as gcp-stackdriver-secret?

Reply:
Done